tests/apply_rules.rs

```diff
@@ -37,9 +37,7 @@ fn test_apply_rules(
             "trips.txt",
             "stops.txt",
             "networks.txt",
-            "admin_stations.txt",
             "ticket_use_perimeters.txt",
-            "admin_stations.txt",
```

Comment:
It looks like you removed all `admin_stations.txt` from the fixtures. But maybe we should keep at least one to check we're keeping this file and the information in it?

Reply:
no need for this test